lib/checkMergeConflicts.js

```diff
@@ -108,3 +114,49 @@ module.exports.checkMergeConflictsInAllPullRequests = async function(context) {
     this.checkMergeConflictsInPullRequest(context, pullRequestDetails);
   }
 };
+
+/**
+ * Ping all pull requests to merge from develop if a PR that requires it got merged.
+ * @param {import('probot').Context} context
+ */
+module.exports.pingAllPullRequestsToMergeFromDevelop = async function (
+  context
+) {
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse} pullRequest
+   */
+  const mergedPR = context.payload.pull_request;
+  const prLabels = mergedPR.labels.map((label) => label.name);
+  // Check if the PR that got merged requires all other PRs to be updated.
+  if (prLabels.includes(updateAllPRsLabel)) {
+    // Fetch all open pull requests.
+    const pullRequestsResponse = await context.github.pulls.list(
+      context.repo({
+        per_page: 60,
+        state: 'open',
+      })
+    );
+
+    const allPullReqests = pullRequestsResponse.data;
+    allPullReqests.forEach(async (pullRequest) => {
+      // Comment on all pull requests.
+      const commentParams = context.repo({
+        body:
+          'Hi @' +
+          pullRequest.user.login +
+          ', there is a new change in develop which needs to ' +
```

Comment:
Link to the change.

Reply:
Also, I would suggest to update our wiki - the section on labels to explain this label and send a note to oppia-dev.

Reply:
> Link to the change.

Done.

> Also, I would suggest to update our wiki - the section on labels to explain this label and send a note to oppia-dev.

Yes. This will be done after the PR is merged.